main.go

```diff
@@ -173,6 +178,11 @@ func main() {
 		recipeCmd.Flags().BoolVar(&bindExternal, "bind-external", false, "bind host ports to external interface")
 		recipeCmd.Flags().BoolVar(&withPrometheus, "with-prometheus", false, "whether to gather the Prometheus metrics")
 		recipeCmd.Flags().StringVar(&networkName, "network", "", "network name")
+		recipeCmd.Flags().BoolVar(&k8sFlag, "k8s", false, "Generate Kubernetes manifests")
```

Comment:
These were the things I wanted to avoid for the integration. K8s must be a separated lifecycle from the normal playground. Playground should generate an artifacts folder which gets consumed by the k8s generator in a second step.

Reply:
I'm unsure if I can follow you. The idea of the `--k8s` flag is to only use it with dry run, i.e.
```
$ playground cook l1 --dry-run --k8s --output ./output --storage-type local-path
```
What the flag does is create a k8s-manifest.yaml, which will then be consumed by the k8s generator/operator.
We need to convert the internal playground manifest to the k8s manifest. We shouldn't work with the docker-compose.yml as that has already abstracted away some of the core information needed.

Reply:
Oh I see, just saw your `manifest.json` patch. You would like the k8s impl to use this manifest.json and not include the k8s manifest logic in playground at all?